.github/workflows/gradle.yml

```diff
@@ -33,65 +33,89 @@ on:
     branches: [ "main" ]
 
 jobs:
-  build:
 
+  unit-tests:
```

Comment:
Need to update `.asf.yaml` as well

Reply:
I think we're going to have a problem to merge this PR because the `build` job is still marked as required, but it doesn't exist anymore.